sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/training/FormTrainingClientBuilder.java

```diff
@@ -207,6 +208,15 @@ public FormTrainingClientBuilder httpLogOptions(HttpLogOptions logOptions) {
         return this;
     }
 
+    /**
+     * Gets the default Azure Form Training client headers and query parameters allow list.
+     *
+     * @return The default {@link HttpLogOptions} allow list.
+     */
+    public static HttpLogOptions getDefaultLogOptions() {
```

Comment:
It's good to include the reason for adding this API in the PR description. Why do we want to add this API?

Reply:
It is better to have default log options methods. @alzimmermsft